src/main/java/com/fasterxml/jackson/databind/util/LRUMap.java

```diff
@@ -25,29 +24,26 @@
     implements LookupCache<K,V>, // since 2.12
         java.io.Serializable
 {
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 2L;
 
-    protected final transient int _maxEntries;
+    protected final int _initialEntries;
+    protected final int _maxEntries;
+    protected final transient ConcurrentLinkedHashMap<K,V> _map;
 
-    protected final transient ConcurrentHashMap<K,V> _map;
-    
     public LRUMap(int initialEntries, int maxEntries)
     {
-        // We'll use concurrency level of 4, seems reasonable
-        _map = new ConcurrentHashMap<K,V>(initialEntries, 0.8f, 4);
+        _initialEntries = initialEntries;
         _maxEntries = maxEntries;
+        // We'll use concurrency level of 4, seems reasonable
```

Comment:
Since Java 8's ConcurrentHashMap, this has no real meaning anymore. You can just ignore that setting.